.github/workflows/master-e2e.yaml

```diff
@@ -28,6 +31,9 @@ on:
         description: Destroy the auto-generated self-hosted runner
         default: true
         type: boolean
+      qase_run_id:
+        description: Case run ID where the results will be reported
+        type: string
```

Comment:
Please add `default: auto` here. It shouldn't hurt, and we will know what value is expected here